src/Features/VisualBasic/Portable/IntroduceVariable/VisualBasicIntroduceParameterService.Rewriter.vb

```diff
@@ -0,0 +1,41 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.IntroduceVariable
+    Partial Friend Class VisualBasicIntroduceParameterService
+
+        Private Class Rewriter
+            Inherits VisualBasicSyntaxRewriter
+
+            Private ReadOnly _replacementAnnotation As New SyntaxAnnotation
+            Private ReadOnly _replacementNode As SyntaxNode
+            Private ReadOnly _matches As ISet(Of ExpressionSyntax)
+
+            Private Sub New(replacementNode As SyntaxNode, matches As ISet(Of ExpressionSyntax))
+                _replacementNode = replacementNode
+                _matches = matches
+            End Sub
+
+            Public Overrides Function Visit(node As SyntaxNode) As SyntaxNode
+                Dim expression = TryCast(node, ExpressionSyntax)
+                If expression IsNot Nothing AndAlso _matches.Contains(expression) Then
+                    Return _replacementNode _
+                        .WithLeadingTrivia(expression.GetLeadingTrivia()) _
+                        .WithTrailingTrivia(expression.GetTrailingTrivia()) _
```

Comment:
place `.` on previous line, so we don't need `_`.

Reply:
WithTriviaFrom

Reply:
Fixed